src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs

```diff
@@ -210,12 +210,24 @@ public void SelectCollectionAndClose(string path)
 			SelectedPath = path;
 			if (!string.IsNullOrEmpty(path))
 			{
+				if (IsInASourceCollectionFolder(path))
+				{
+					var msg = L10NSharp.LocalizationManager.GetString("OpenCreateCloneControl.InSourceCollectionMessage",
+						"This collection is part of your 'Sources for new books' which you can see in the bottom left of the Collections tab. It cannot be opened for editing.");
```

Comment:
While there is, for some reason, a pre-existing  OpenCreateCloneControl localization ID, there is also a better OpenCreateNewCollectionDialog one. Does it make sense to use that instead?